src/components/reps/array-handler.jsx

```diff
@@ -53,12 +53,26 @@ export default {
         <span key="array_elt_empty" title="empty array" />
       )
     }
+
+    if (inContainer) {
+      return (
+        <div className="array-rep">
+          [
+          <ul>
```

Comment:
what is the idea behind the adding the <ul> and the line breaks? I think overall i'd prefer eg
`[42, [32, 5]]` to having it broken out across multiple lines in block level elements.